engines/bops_api/app/services/bops_api/application/search_service.rb

```diff
@@ -76,6 +77,14 @@ def apply_date_filters
         end
       end
 
+      def filter_by_application_decision
+        decision = params[:councilDecision]
+
+        return scope if decision.blank?
+
+        scope.for_council_decision(decision)
+      end
+
```

Comment:
you've caused me to realise i implemented the comma separation thing entirely differently from ben. i guess we have to look at that at some point soon. it is however much simpler than i realised it would be. 🤷🏻